app/Organizer/OrganizerJSONLDServiceProvider.php

```diff
@@ -55,7 +55,8 @@ function () use ($container) {
                     ),
                     new CdbXMLImporter(
                         $container->get(CdbXMLToJsonLDLabelImporter::class)
-                    )
+                    ),
+                    $container->get('config')['completeness']['organizer']
```

Comment:
Don't know about this one,
On the one hand you provide a fallback with an empty array in the config: 
`file_exists(__DIR__ . '/config.completeness.php') ? require __DIR__ . '/config.completeness.php' : [],
`
But if the empty array doesn't contain `organizer`, it could still fail because you get a null. Though I still doubt whether it would be good to provide another fallback or fall hard

Reply:
I'm leaning towards expecting there will always be a completeness config for organizer. So I would not fallback with an empty array, but fail hard instead.

Reply:
See also the discussion here https://github.com/cultuurnet/appconfig/pull/557, I think it's indeed much better to fail hard instead of having unexpected side effects.

Reply:
I agree with the fail hard, just because we always failed softly doesn't mean we should continue with that approach.
This was just a special case where failing soft & failing hard were mixed.